packages/theming/android-theme/src/androidTheme.colors.ts

```diff
@@ -0,0 +1,198 @@
+    disabledText: p.textDisabledHighContrast,
+    disabledBodyText: p.textDisabled,
+    disabledSubtext: p.gray200,
+    disabledBodySubtext: p.gray300,
+
+    focusBorder: 'transparent',
+    variantBorder: 'transparent',
+    variantBorderHovered: 'transparent',
+    defaultStateBackground: 'transparent',
+
+    errorText: p.dangerPrimary,
+    warningText: p.warningPrimary,
+    errorBackground: p.dangerTint10,
+    blockingBackground: p.dangerTint10,
+    warningBackground: p.warningPrimary,
+    warningHighlight: p.warningTint10,
+    successBackground: p.successTint10,
+
+    inputBorder: 'transparent',
+    inputBorderHovered: 'transparent',
+    inputBackground: 'transparent',
+    inputBackgroundChecked: 'transparent',
+    inputBackgroundCheckedHovered: 'transparent',
+    inputForegroundChecked: p.communicationBlue,
+    inputFocusBorderAlt: 'transparent',
+    smallInputBorder: 'transparent',
+    inputText: p.textPrimary,
+    inputTextHovered: p.textPrimary,
+    inputPlaceholderText: p.textSecondary,
+
+    buttonBackgroundChecked: p.buttonBackground,
+    buttonBackgroundHovered: p.buttonBackground,
+    buttonBackgroundCheckedHovered: p.buttonBackground,
+    buttonBackgroundPressed: p.buttonBackgroundPressed,
+    buttonBackgroundDisabled: p.buttonBackgroundDisabled,
+    buttonText: p.textOnAccent,
+    buttonTextHovered: '#ffffff',
+    buttonTextChecked: '#ffffff',
+    buttonTextCheckedHovered: '#ffffff',
+    buttonTextPressed: p.textOnAccent,
+    buttonTextDisabled: p.buttonTextDisabled,
+    buttonBorderDisabled: 'transparent',
+    buttonBorderFocused: 'transparent',
+
+    primaryButtonBackground: p.buttonBackground,
+    primaryButtonBackgroundHovered: p.buttonBackground,
+    primaryButtonBackgroundPressed: p.buttonBackgroundPressed,
+    primaryButtonBackgroundDisabled: p.buttonBackgroundDisabled,
+    primaryButtonBorder: 'transparent',
+    primaryButtonBorderFocused: 'transparent',
+    primaryButtonText: p.textOnAccent,
+    primaryButtonTextHovered: '#ffffff',
+    primaryButtonTextPressed: '#ffffff',
+    primaryButtonTextDisabled: p.buttonTextDisabled,
+
+    accentButtonBackground: p.buttonBackground,
+    accentButtonText: p.textOnAccent,
+
+    menuBackground: p.menuBackground,
+    menuDivider: 'transparent',
+    menuIcon: p.menuIcon,
+    menuHeader: 'transparent',
+    menuItemBackgroundHovered: 'transparent',
+    menuItemBackgroundPressed: 'transparent',
+    menuItemText: p.menuItemText,
+    menuItemTextHovered: p.gray900,
+
+    listBackground: p.listBackground,
+    listText: p.menuItemText,
+    listItemBackgroundHovered: 'transparent',
+    listItemBackgroundChecked: 'transparent',
+    listItemBackgroundCheckedHovered: 'transparent',
+
+    listHeaderBackgroundHovered: 'transparent',
+    listHeaderBackgroundPressed: 'transparent',
+
+    actionLink: 'todo',
+    actionLinkHovered: 'todo',
+    link: 'todo',
+    linkHovered: 'todo',
+    linkPressed: 'todo',
```

Comment:
If you don't have every color defined just yet, you can specify this as a partial palette and leave this out. But keeping a string 'todo' likely will just cause the app to crash/error if it ever shows a link while using the android theme.

Reply:
https://github.com/microsoft/fluentui-react-native/blob/03896e322970c7f2c052ddf1ea8be8dfac6e4e7e/packages/theming/theme-types/src/Color.types.ts#L454